container/container.go

```diff
@@ -56,6 +56,10 @@ func StopContainer(client DockerClient, containerId string, timeout uint) error
 		return err
 	}
 
+	if cntr.State.Status == "exited" {
```

Comment:
It's `exited `, used a (few lines bellow](https://github.com/Nitro/sidecar-executor/blob/de53bdc593da4085b4fc30540c9fea8d8896dcd5/container/container.go#L64)